Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Controllers/UsersController.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Know_Your_Nation_Speedy.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Storage;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Know_Your_Nation_Speedy.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UsersController : ControllerBase
+    {
+        private readonly MyDbContext _db;
+        readonly IConfiguration _config;
+        public UsersController(MyDbContext context, IConfiguration config)
+        {
+            _db = context;
+            _config = config;
+        }
+        // GET api/values
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Users>>> Get()
+        {
+            return await _db.UsersEntries.ToListAsync();
+        }
+        // GET api/values/5
+        [HttpGet("{id}")]
+        public async Task<IActionResult> GetEntry([FromRoute] int id)
+        {
+            var entry = await _db.UsersEntries.SingleOrDefaultAsync(m => m.UsersId == id);
+            if (entry == null)
+            {
+                return NotFound();
+            }
+            await _db.SaveChangesAsync();
+            return Ok(entry);
+        }
+        // POST api/values
+        [HttpPost]
+        public async Task Post([FromBody] Users value)
```

Comment:
Try to rename "value" to something more appropriate. When you're accessing this method from elsewhere and see that the function takes in "value" it might not make that much sense. Not necessary to change for this PR but good to remember for the future

Reply:
Im aware of the controller being poor, only using it for testing purposes, next sprint task is too add and clean up Controllers, thanks for the feedback